store-front/src/design-system/molecules/SignInModal/SignInModal.tsx

```diff
@@ -124,7 +124,7 @@ export const SignInModal: FC<SignInModalProps> = ({
             input,
         ].join(' ');
 
-        const bytes = '05' + char2Bytes(formattedInput);
+        const bytes = '0501000000' + char2Bytes(formattedInput);
```

Comment:
I guess this is a bytes representation of a very simple something else (like integer '1'?). If so I'd prefer a ...2Bytes(<the real underlying thing>) over this bytes string constant